osu.Game/Screens/Play/EpilepsyWarning.cs

```diff
@@ -18,11 +16,7 @@ namespace osu.Game.Screens.Play
 {
     public class EpilepsyWarning : VisibilityContainer
     {
-        public const double FADE_DURATION = 500;
-
-        private readonly BindableDouble trackVolumeOnEpilepsyWarning = new BindableDouble(1f);
-
-        private Track track;
+        public const double FADE_DURATION = 250;
```

Comment:
was this change for stylistic reasons?

Reply:
Correct, 500ms fadeout on transition to player looked weird to me.

Reply:
Guess it's fine. I set it higher for the inwards transition specifically, but not a huge deal.